verilog/formatting/tree_unwrapper_test.cc

```diff
@@ -1643,7 +1643,7 @@ const TreeUnwrapperTestData kClassTestCases[] = {
         ClassHeader(0, L(0, {"class", "c", ";", "// c is for cookie"})),
         ClassItemList(1, L(1, {"// f is for false"}),
                       FunctionHeader(1, L(1, {"function", "f", "("}),
-                                     TFPortList(3, L(3, {"integer", "size"})),
+                                     TFPortList(2, L(2, {"integer", "size"})),
```

Comment:
As mentioned in formatter_test.cc, these should wrap +2 levels (= +4 spaces).

Reply:
Fixed